bundles/org.openhab.binding.samsungtv/src/main/resources/OH-INF/i18n/samsungtv_hu.properties

```diff
@@ -209,6 +209,7 @@ channel-type.samsungtv.keycode.state.option.KEY_MORE = KEY_MORE
 channel-type.samsungtv.keycode.state.option.KEY_MOVIE1 = KEY_MOVIE1
```

Comment:
This file should be removed as i18n is handled by crowdin. Only the default language should be provided.

Somehow i cannot place a commen on an empty file, but the samsungtv_de.properties file should also be removed (it is now empty)

Reply:
Removed